doc: 3.0.0-preview2 doc updates (#21138)
nrfconnect/sdk-nrf: doc/nrf/index.rst

Introduction
############

.. important::

>    |NCS| v3.0.0-preview2 is a development tag used for development purposes but is not recommended to use in a production environment.

Reviewer: ```suggestion
   |NCS| v3.0.0-preview2 is a development tag used for development purposes.
   It is not recommended to be used in a production environment.
```
